src/components/App.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
+import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
 import Routes from '../routes/AppRouter';
+import { useSetUser } from '../store/hooks';
+
+import store from '../store';
 
 const App = () => {
+  useSetUser({ ...store });
   return (
-    <Router>
-      <Routes />
-    </Router>
+    <Provider store={store}>
+      <Router>
```

Comment:
This would not be moved, for the following reasons. 
We need a component called getUser which is already been built by @kechyy at this point to restore the user when a refresh occurs. Also we need to separate concerns across the application to improve a single functionality.

Reply:
As @codexempire mentioned, the TTL suggested these changes since the App and Router are simple components. Kindly revert back to what we have as @kechyy also have this changes.